02-express-tutorial/routes/auth.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+
+
+app.post('/',(req,res)=>{
```

Comment:
```suggestion
app.post('/', (req,res) => {
```

Some more formatting items:

- Always put a space after commas in function calls & arrays
- Put spaces around the arrow (it's considered an operator just like `+`, `-`, `=`, etc.